### PR TITLE
feat: add foreign key in table users

### PR DESCRIPTION
**Pull request by ZulfanoRp**

### CODE REVIEW

File: app/Http/Controllers/DashboardController.php [hunk 1]
```diff
@@ -26,4 +27,8 @@ public function artist()
         $data['navbarType'] = "side";
```
Comment (KrishnaHendra) [inline app/Http/Controllers/DashboardController.php:27]: tanda petik dipodono kabeh nggawe petik dua ae col mariki, ojo bedo"

File: app/Http/Controllers/DashboardController.php [hunk 2]
```diff
@@ -26,4 +27,8 @@ public function artist()
...
         return view('pages.artist', $data);
     }
+
+    // public function test() {
+    //     $users = DB::table('users')
+    // }
```
Comment (KrishnaHendra) [inline app/Http/Controllers/DashboardController.php:33]: lek kate push seng di comment" dihapus ae cek rapi

File: app/Http/Controllers/DashboardController.php (hunk 1 above)
Comment (ZulfanoRp) [reply app/Http/Controllers/DashboardController.php:27]: oh oke

File: app/Http/Controllers/DashboardController.php (hunk 2 above)
Comment (ZulfanoRp) [reply app/Http/Controllers/DashboardController.php:33]: note

File: app/Models/User.php [hunk 3]
```diff
@@ -12,33 +12,26 @@ class User extends Authenticatable
...
     ];
 
-    /**
-     * The attributes that should be hidden for serialization.
-     *
-     * @var array<int, string>
-     */
     protected $hidden = [
         'password',
         'remember_token',
```
Comment (KrishnaHendra) [inline app/Models/User.php]: remember token?

**Review by KrishnaHendra — APPROVED** [review]

**Review by KrishnaHendra — APPROVED** [review]

**Pull request merged**